site/index.html

```diff
@@ -5,37 +5,48 @@
-                  <pre><span class='hs-definition'>primes</span> <span class='hs-keyglyph'>=</span> <span class='hs-varid'>filterPrime</span> <span class='hs-keyglyph'>[</span><span class='hs-num'>2</span><span class='hs-keyglyph'>..</span><span class='hs-keyglyph'>]</span>
+            <div class="row" id="code-example">
+               <div class="branding sample">
+                  <br class="visible-xs visible-sm">
+                  <h4 class="tag">Declarative, statically typed code.</h4>
+                  <div title="This example is contrived in order to demonstrate what Haskell looks like, including: (1) where syntax, (2) enumeration syntax, (3) pattern matching, (4) consing as an operator, (5) list comprehensions, (6) infix functions. Don&#39;t take it seriously as an efficient prime number generator." class="code-sample">
+                     <pre><span class='hs-definition'>primes</span> <span class='hs-keyglyph'>=</span> <span class='hs-varid'>filterPrime</span> <span class='hs-keyglyph'>[</span><span class='hs-num'>2</span><span class='hs-keyglyph'>..</span><span class='hs-keyglyph'>]</span>
   <span class='hs-keyword'>where</span> <span class='hs-varid'>filterPrime</span> <span class='hs-layout'>(</span><span class='hs-varid'>p</span><span class='hs-conop'>:</span><span class='hs-varid'>xs</span><span class='hs-layout'>)</span> <span class='hs-keyglyph'>=</span>
           <span class='hs-varid'>p</span> <span class='hs-conop'>:</span> <span class='hs-varid'>filterPrime</span> <span class='hs-keyglyph'>[</span><span class='hs-varid'>x</span> <span class='hs-keyglyph'>|</span> <span class='hs-varid'>x</span> <span class='hs-keyglyph'>&lt;-</span> <span class='hs-varid'>xs</span><span class='hs-layout'>,</span> <span class='hs-varid'>x</span> <span class='hs-varop'>`mod`</span> <span class='hs-varid'>p</span> <span class='hs-varop'>/=</span> <span class='hs-num'>0</span><span class='hs-keyglyph'>]</span></pre>
+                  </div>
                </div>
             </div>
             <div class="row" id="get-started-button">
                <div class="span12 col-sm-12" style="padding-top: 10px;">
                   <br/>
                   <p class="text-center">
-                     <a href="/downloads/" class="btn btn-lg btn-primary" style="font-size: 1.5em">Get Haskell</a>
+                     <a href="/get-started/" class="btn btn-lg btn-primary" style="font-size: 1.5em">GET STARTED</a>
                   </p>
                </div>
             </div>
          </div>
       </div>
    </div>
 </div>
-<br><br class="hidden-xs hidden-sm"><br class="hidden-xs hidden-sm">
+
+<br>
+<br class="hidden-sm">
+<br class="hidden-sm">
+<br class="hidden-xs hidden-sm">
+<br class="hidden-xs hidden-sm">
```

Comment:
Here I added 2 `<br class="hidden-sm">`, which add some extra space on the small monitors. This is for the same purpose as the other `<br>`s I added -> to center the get-started button in the page and push Videos section a bit lower.